pytorch_lightning/trainer/evaluation_loop.py

```diff
@@ -329,9 +329,9 @@ def store_predictions(self, output, batch_idx, dataloader_idx):
     def on_evaluation_epoch_end(self, *args, **kwargs):
         # call the callback hook
         if self.testing:
-            self.trainer.call_hook('on_test_epoch_end', *args, capture=True, **kwargs)
+            self.trainer.call_hook('on_test_epoch_end', *args, **kwargs)
         else:
-            self.trainer.call_hook('on_validation_epoch_end', *args, capture=True, **kwargs)
+            self.trainer.call_hook('on_validation_epoch_end', *args, **kwargs)
```

Comment:
same as last comment:

```suggestion
        self.trainer.call_hook('on_test_epoch_end' if self.testing else 'on_validation_epoch_end', *args, **kwargs)
```

Reply:
Same here